src/cli/load.js

```diff
@@ -5,12 +5,29 @@ import type {Command} from "./command";
 import {loadInstanceConfig, pluginDirectoryContext} from "./common";
 import {LoggingTaskReporter} from "../util/taskReporter";
 import * as pluginId from "../api/pluginId";
+import {Plugin, PluginDirectoryContext} from "../api/plugin";
```

Comment:
By the way: when you only use imported symbols in type positions rather
than runtime positions—i.e., when the imports are unused after removing
Flow type annotations—please use `import type` rather than `import`.
This prevents adding an unnecessary runtime dependency edge.